### DISCUSSION TITLE
Vertically flip image texture

### DISCUSSION THREAD

**Discussion by felipeleivav** [google/model-viewer#4240] [Q&A]
Hi! I'm loading a texture on a model but before this I want to vertically flip my image.

I read that there's a flipY property for textures, but looking at the code and console outputs I'm still not sure about how to use it.

Here's my code:

```
const image = this.canvas.toDataURL('png');
const modelViewer = document.querySelector('#mug');

modelViewer.addEventListener('load', () => {
  modelViewer.createTexture(image).then(texture => {
    const material = modelViewer.model.materials[2];
    material.pbrMetallicRoughness.baseColorTexture.setTexture(texture);
    modelViewer.cameraOrbit = '0deg 90deg 100%';
    modelViewer.fieldOfView = 'auto';
  });
});
```

Since I'm loading from a data URL I think it's not possible to flip the image, so I would ideally prefer to flip texture instead, but not really sure if this is possible.

Thanks!

**Comment by elalish** [comment]
Once https://github.com/google/model-viewer/pull/4209 lands, I believe that will work.
